cmd/options.go

```diff
@@ -142,6 +144,10 @@ func Parse() *Options {
 			}
 			log.Logger = log.Output(consoleWriter)
 
+			if opts.DryRun {
+				return nil
+			}
+
```

Comment:
```suggestion
```
I think adding this would prevent all the `✨ Running %s (%s) with` comments from being printed in `--dry-run` mode - but I want them to _always_ print for consistency.